apps/etterlatte-tidshendelser/src/main/resources/db/migration/V11__aktivitetsplikt_varigunntak_6mnd_juli_august.sql

```diff
@@ -0,0 +1,5 @@
+-- Jobben for å vurdere aktivitetsplikt etter 6 mnd
+INSERT INTO jobb (type, kjoeredato, behandlingsmaaned)
+values ('OMS_DOED_6MND_INFORMASJON_VARIG_UNNTAK', '2024-07-23', '2024-07');
```

Comment:
```suggestion
values ('OMS_DOED_6MND_INFORMASJON_VARIG_UNNTAK', '2024-08-23', '2024-07');
```
ellers så kjører den vel ikke i dag?

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/5596/commits/a024c87ac10668b1e2192ee66340a8bd3902dfb9